include/render/gles2.h

```diff
@@ -17,7 +17,6 @@
 struct wlr_gles2_pixel_format {
 	uint32_t drm_format;
 	GLint gl_format, gl_type;
-	int depth, bpp;
 	bool has_alpha;
```

Comment:
Can't we remove this one as well? It seems like it's in the shared table now.

Reply:
It's required to identify if the format is opaque or not. Otherwise we are not able to tell the difference between ARGB and XRGB because they have the same `gl_format` and `gl_type` value.

Reply:
Yes, but we have `has_alpha` in `wlr_pixel_format_info` as well. Why do we need to duplicate it in `wlr_gles2_pixel_format` as well?

Reply:
It's a search parameter in `get_gles2_format_from_gl`,  we might not get the good drm format without it.

Reply:
Oh. That's unfortunate indeed. We could query the shared table inside the `get_gles2_format_from_gl` loop, but that's not great either.

Let's just leave it at that for now then.